docs/guides/user-guide/general-faqs.md

```diff
@@ -0,0 +1,189 @@
+
+If you're interested in Teams Beta, come onboard with us for a private technology preview of this plan! You can sign up [here](https://stackblitz.com/beta-teams-signup).
+
+## Enterprise Server
+
+### What is the Enterprise Server offering?
+
+Enterprise Server is a membership offering that's best fit for organizations who want to use StackBlitz behind their firewall. Benefits include:
+
+- Private NPM registry integration
+- Custom SSO integration
+- On-prem installation support
+- Dedicated Customer Success Engineer
+
+Check out the [Enterprise page](https://stackblitz.com/enterprise) for details and set up a call with our Enterprise team!
+
+### I want to purchase the enterprise edition for WebContainer. Where can I do that?
+
+Bringing Webcontainer to Enterprise Server is in the works, but not currently available. You can reach out to enterprise@stackblitz.com for details.
+
+We do offer [WebContainer API](webcontainers.io) licenses for organizations wanting to use WebContainer technology for production usage in a for-profit setting. Check out [webcontainers.io](webcontainers.io) for more information!
+
+# Technology FAQs
+
+## Classic Editor (Engineblock projects)
+
+### What is the Classic Editor for Engineblock projects?
+
+The Classic Editor utilizes Engineblock and WeBcontainer technologies to run. As the original StackBlitz interface, the Classic Editor has a file tree, an npm package manager, and a light dev server and preview pane that allows you to see your app as you code.
+
+Engineblock projects are distinguishable as they do not have a terminal.
+
+
+### How do I create code snippets?
+
+Code snippets are available in our Classic Editor for both Engineblock and WebContainer projects. Click on the Settings icon in the activity bar to the left of your screen. The Settings sidebar will pop open and you will see User snippets and Workspace snippets. User snippets are available for any of your projects. Workspace snippets are only available in the current project you create them in.
+
+
+### Can I change the TypeScript version in my project?
+
+The current TypeScript starter uses EngineBlock technology, which has a fixed TypeScript version. EngineBlock is great for lightweight frontends, but does not run Node.js itself and will be limited to things like TypeScript versioning.
+
+Webcontainer, however does run Node.js itself in the browser and does not have this limitation. You can simply `npm install` any TypeScript version as you would locally.
+
+## Classic Editor (WebContainer projects)
+
+### What is the Classic Editor for WebContainer projects?
+
+The Classic Editor utilizes Engineblock and WeBcontainer technologies to run. As the original StackBlitz interface, the Classic Editor has a file tree, an npm package manager, and a light dev server and preview pane that allows you to see your app as you code.
```

Comment:
rewrite sentence - reads as if BOTH technologies are needed to run classic editor, but not true.